### ISSUE TITLE
New Logo Design

### ISSUE THREAD

**Issue by neemanbhat** [fenimore/DemocracyDroid#47]
Hi sir, I will to design a new logo for your app. You can tell me about the requirements if it may please you.

**Comment by fenimore** [comment]
Absolutely! This would be great

The logo should be red, like Democracy Now!'s logo -- and I quite like the "Sabot" wooden clog as the icon, because it's representative of Sabotage (and early labor struggles against industrialization) 

But if somethign inspires you, go for it and we'll see 🤔 ? One thing, don't make it look like Democracy Now's logo though, because of copyright etc etc 😳 😄

**Comment by neemanbhat** [comment]
Hahaha exactly i will do my best and consider these things too.

**Comment by neemanbhat** [comment]
Internet is down for few days i will get back to you asap.

**Comment by fenimore** [comment]
no rush :)

**Comment by neemanbhat** [comment]
Apologies for not being able to revert back soon as the Internet was snapped.

**Comment by fenimore** [comment]
Hi @neemanbhat thanks for your effort. This is a really cool and symbolically charged idea -- however I think it is too busy for what we're trying to go for with Democracy Droid!. The app itself more or less follows the material design manual. I think something up that alley would jive better. 

Do you see what I mean by that? Have you ever seen [this](https://romannurik.github.io/AndroidAssetStudio/icons-launcher.html#foreground.type=clipart&foreground.clipart=android&foreground.space.trim=1&foreground.space.pad=0.25&foreColor=rgba(96%2C%20125%2C%20139%2C%200)&backColor=rgb(68%2C%20138%2C%20255)&crop=0&backgroundShape=square&effects=none&name=ic_launcher) for example?

**Comment by neemanbhat** [comment]
Okay I do see what you mean by that. Let me try again and come up with something close to this.

**Comment by neemanbhat** [comment]
![img-20180520-wa0015](https://user-images.githubusercontent.com/34791884/40280242-102b4b42-5c6e-11e8-9cce-e8587e945f53.jpg)
Check this

**Comment by fenimore** [comment]
Thank you @neemanbhat 
I'll consider using this logo but for the moment we'll stick with the pre-existing logo
